forc-plugins/forc-client/src/cmd/call.rs

```diff
@@ -0,0 +1,176 @@
+use crate::NodeTarget;
+use clap::Parser;
+use either::Either;
+use fuel_crypto::SecretKey;
+use fuels::programs::calls::CallParameters;
+use fuels_core::types::{AssetId, ContractId};
+use std::{path::PathBuf, str::FromStr};
+use url::Url;
+
+pub use forc::cli::shared::{BuildOutput, BuildProfile, Minify, Pkg, Print};
+pub use forc_tx::{Gas, Maturity};
+
+forc_util::cli_examples! {
```

Comment:
For some reason these examples aren't showing up when I do `forc call --help`. Here's what I did to get it to build if you want the changes: https://github.com/FuelLabs/sway/pull/6831

Installed with `cargo install --path forc-plugins/forc-client`

Reply:
thanks for this!